catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/service/SearchService.java

```diff
@@ -84,7 +85,7 @@
 
     private static final long DEFAULT_TIMEOUT = 300000;
 
-    private static final int DEFAULT_COUNT = 10;
+    private static final long DEFAULT_COUNT = 10L;
```

Comment:
`DEFAULT_COUNT` is only being used once, to set local long value `count` in **CreateQuery()**. `count` is then used to create a **QueryImpl** object, which is getting converted back into an int (315). Maybe we should have `DEFAULT_COUNT` and `count` as integers, and on line 276 we just convert `countLng` to an int.